src/test/ui/asm/inline-syntax.rs

```diff
@@ -0,0 +1,23 @@
+#![feature(asm, llvm_asm)]
```

Comment:
This test needs to be conditional on target, or specify via `compiler-flags` what target to use. In later case, you will also need to make the test `no_core` – you can look at the other assembly tests (such as `src/test/assembly/asm/x86-types.rs`) to see how to set that up.

Reply:
The test should also check the behaviour when the target isn't x86_64.